src/main/java/com/ericsson/ei/handlers/EventToObjectMapHandler.java

```diff
@@ -81,30 +81,36 @@ public void setJmesPathInterface(JmesPathInterface jmesPathInterface) {
         return getEventToObjectList(eventId);
     }
 
+    /**
+     * To check and save the eventIds to the objectId in the mapped database.
+     * @param rulesObject
+     * @param event
+     * @param objectId aggregated event object Id
+     */
     public void updateEventToObjectMapInMemoryDB(RulesObject rulesObject, String event, String objectId) {
         String eventId = getEventId(rulesObject, event);
-        String condition = "{\"_id\" : \"" + eventId + "\"}";
-        ArrayList<String> list =  getEventToObjectList(eventId);
-        boolean firstTime = list.isEmpty();
-        list = updateList(list, eventId, objectId);
-        ObjectMapper mapper = new ObjectMapper();
-        JsonNode entry = null;
-
+        String condition = "{\"_id\" : \"" + objectId + "\"}";
```

Comment:
With the structure of this document now changed, and the waitlist affected, some tests are probably in need of updates also.

Reply:
No it will not be affected as the query mechanism from the waitlist handler has been changed to search instead in the events (previously objects)